phpBB/adm/style/acp_search_index.html

```diff
@@ -0,0 +1,80 @@
+{% include 'overall_header.html' %}
+
+<a id="maincontent"></a>
+
+<script>
+// <![CDATA[
+	/**
+	* Popup search progress bar
+	*/
+	function popup_progress_bar(progress_type)
+	{
+		close_waitscreen = 0;
+		// no scrollbars
+		popup('{{ UA_PROGRESS_BAR }}&amp;type=' + progress_type, 400, 240, '_index');
+	}
+// ]]>
+</script>
+
+<h1>{{ lang('ACP_SEARCH_INDEX') }}</h1>
+
+<p>{{ lang('ACP_SEARCH_INDEX_EXPLAIN') }}</p>
+
+{% for backend in loops.backend %}
```

Comment:
`loops.` are obsoleted since 3.2.x

Reply:
I followed the guide in the forum
https://www.phpbb.com/community/viewtopic.php?t=2424606
how is done now?

Reply:
`{% for backend in backend %}` is acceptable.
It yould be better to have something like `{% for backend in backends %}` though.